test/model.validate.test.js

```diff
@@ -144,4 +144,26 @@ describe('model: validate: ', function() {
     const err2 = await User.validate({ name: 'Sam' }).then(() => null, err => err);
     assert.ok(err2 === null);
   });
+  it('Model.validate(...) supports passing in an object, array or string (gh-10353)', async function() {
+    const testSchema = new Schema({
+      docs: String,
+      name: String,
+      age: Number,
+      weight: Number
+    });
+
+    const Test = mongoose.model('Test', testSchema);
+
+    const test = { name: 1, docs: 'a doc' };
+    let pathsOrOptions = { pathsToSkip: ['name'] };
+
+    const err = await Test.validate(test, pathsOrOptions);
```

Comment:
This test case is severely lacking because none of the properties have any validators, so of course `validate()` will always pass. Instead, please add a validator on each path that always fails, like `validate: () => false`, and test that `validate()` returns the correct errors with each form of `Test.validate()`.